prometheus/rootfs/bin/boot.sh

```diff
@@ -22,18 +22,23 @@ done
 
 set +e
 
-OPTS=""
+until [ -n "${DEIS_MONITOR_ALERT_SERVICE_HOST+1}" ]; do
```

Comment:
nah that totally makes sense.  it's just weird for the service discovery stuff to have an irrecoverable race condition.